### PR TITLE
Stop Mantaro from throwing an error on boot if no Weeb API key is set

### PR DESCRIPTION
**Pull request by netux**
If "weebapiKey" was set to null in the config, the bot would crash as it requests and parses some JSON from Weeb API but the API returned an error instead.

I didn't read through the code too much to find a better way to do this. If there is let me know.

### CODE REVIEW

**Comment by Kodehawa** [comment]
As far as am aware, the current behaviour is exactly the same- it throws up and exception and doesn't register the commands, but it continues booting like so. I just tested it locally without an API key (aka how I always run my local instance) and the bot started up fine for me :thinking:

**Comment by netux** [comment]
Strange, I remember it crashing last time I tried but I just tested as well and it works fine 😅.
Well, this at least stops the error from spamming the console, so I'll rename appropriately.

**Comment by Xernium** [comment]
But this is by design isnt it? The module is supposed to shut down if it doesnt get a valid API-Key

File: src/main/java/net/kodehawa/mantarobot/commands/ActionCmds.java [hunk 1]
```diff
@@ -137,18 +66,102 @@ public void register(CommandRegistry cr) {
...
         cr.register("meow", new ImageCmd("Meow", "Meows at the specified user.", "meow", MEOW.get(), "commands.action.meow"));
         cr.registerAlias("meow", "mew");
 
-        //nom();
-        cr.register("nom", new ImageCmd("Nom", "*nom nom*", "nom", "nom", "commands.action.nom"));
-
-        //facedesk();
-        cr.register("facedesk", new ImageCmd("Facedesk", "When it's just too much to handle.", "facedesk", "banghead",
-                "commands.action.facedesk", true));
+        if (MantaroData.config().get().hasWeebapiKey()) {
```
Comment (Xernium) [inline src/main/java/net/kodehawa/mantarobot/commands/ActionCmds.java:73]: If you're going here an .isValid() method would also be a good idea. Additionally, one could check wether the API key has the required Permissions on Weeb.sh

Comment (Xernium) [reply src/main/java/net/kodehawa/mantarobot/commands/ActionCmds.java:73]: Refer to https://docs.weeb.sh/

**Comment by Kodehawa** [comment]
I'm not sure if this is the best way to achieve this- at the end, the commands don't register eitherway, and it just gives you a little warning. 

I guess I can leave it up to discussion.

**Comment by netux** [comment]
There are still some commands in the module that use a hard-coded list instead of the WeebAPI, which won't get registered if the module crashes on load.

On the other hand, most likely only devs or people trying to setup their own version of the bot will see it 😛.

I'll close the PR as you guys don't like this method and it seems there is no real need to fix the crash anyway.

**Pull request closed without merging**